lib/libc/minimal/source/stdlib/abort.c

```diff
@@ -11,4 +11,9 @@ void abort(void)
 {
 	printk("abort()\n");
 	k_panic();
+#ifdef __has_builtin
+#if __has_builtin(__builtin_unreachable)
+	__builtin_unreachable();
+#endif
+#endif
```

Comment:
This can be replaced with the toolchain abstraction layer `CODE_UNREACHABLE` macro.

Reply:
thanks, will do